x-pack/plugins/enterprise_search/public/applications/shared/layout/page_template.tsx

```diff
@@ -61,6 +63,8 @@ export const EnterpriseSearchPageTemplate: React.FC<PageTemplateProps> = ({
   const hasCustomEmptyState = !!emptyState;
   const showCustomEmptyState = hasCustomEmptyState && isEmptyState;
 
+  appendBetaNotificationItem(solutionNav);
```

Comment:
see commit note about mutating the array. tl;dr it creates a bunch of annoying type errors when not mutating so it's easier to do this for now.